docs/api/services.md

```diff
@@ -0,0 +1,308 @@
+---
+id: services
+title: 🥡 Services
+sidebar_position: 9
+---
+
+Services provide external services needed to run the application while developing it. They are supposed to be used while developing and testing the application.
```

Comment:
The first sentence can remove "while developing it" since the 2nd one already mentions it.